### ISSUE TITLE
Automatically played 'next episode' is not shown in JellyStat

### ISSUE THREAD

**Issue by jbeige** [CyferShepard/Jellystat#157]
**Describe the bug**
JellyStat only shows episodes of a show that are manually played. When watching a series Jellyfin will automatically show the next episode unless cancelled, and these additional episodes are not showing up in JellyStat.

**Environment Details (please complete the following information):**
 - OS: Ubuntu
 - Browser: Safari (mobile) and Vivaldi (desktop). Media was played in the Jellyfin Media Player
 - Jellystat Version 1.0.9
 - Jellyfin Version 10.8.13

**To Reproduce**
1. Watch an episode from a series
2. When the episode ends, allow jellyfin to automatically play the next episode
3. JellyStat only shows the first episode as being played.

**Expected behavior**
All episodes that are watched should show up in JellyStat

**Comment by CyferShepard** [comment]
Hey @jbeige , im trying to replicate this but i cant seem to do so yet, just wanted to confirm, are you using the stable or unstable build?

**Comment by jbeige** [comment]
I am using the stable build, installed with docker compose. I can send logs or anything that you might need.On Jan 11, 2024, at 2:43 AM, Thegan Govender ***@***.***> wrote:﻿
Hey @jbeige , im trying to replicate this but i cant seem to do so yet, just wanted to confirm, are you using the stable or unstable build?

**Comment by CyferShepard** [comment]
Thats alright, there wont be any logs regarding whats inserted, il just have to try and replicate these actions. Are there any special conditions setup on your media? Eg All episodes of all seasons exist in a root folder and not sub folders by season? Just need to try and setup a similar environment on my side to replicate the issue

**Comment by jbeige** [comment]
Yes, all episodes are in a root folder for each show. Other than that I don’t think any special configurations exist.On Jan 11, 2024, at 6:45 AM, Thegan Govender ***@***.***> wrote:﻿
Thats alright, there wont be any logs regarding whats inserted, il just have to try and replicate these actions. Are there any special conditions setup on your media? Eg All episodes of all seasons exist in a root folder and not sub folders by season? Just need to try and setup a similar environment on my side to replicate the issue

**Comment by dandud100** [comment]
I have the same issue (using unstable)

**Comment by anshuman852** [comment]
i have the same problem
using latest stable

**Comment by ShakeSp33r** [comment]
Same problem here, activity recording is also very inconsistent. 

**I'm currently using :**
Jellyfin 10.9.7 (2024/06/25)
Jellystat Unstable (2024/06/22)

The episode/movie being watch do show up in the Session section of the home page, but not always in the activity section. Might watch 4 episodes back to back and only first & last episode show in activity window. It is also not always 1st & 3rd, it could be any combination.

**Comment by llama47** [comment]
Also experiencing this issue on latest stable versions:
-Jellystat 1.1.0
-Jellyfin 10.9.7

**Comment by dandud100** [comment]
#221

**Comment by ShakeSp33r** [comment]
Some more testing over the weekend...(Using the latest Unstable Ver)

When leaving Android TV to play through the credits and start the next episode everything seems to record correctly, but when you click the next button ![Screenshot 2024-08-12 at 08 37 21](https://github.com/user-attachments/assets/4457d2ef-3e73-49a5-b9ab-46bd67bba885) in the player, it was not added.
